addons/stock/models/stock_picking.py

```diff
@@ -1889,22 +1890,21 @@ def calculate_date_category(self, datetime):
         start_day_2 = start_today + timedelta(days=2)
         start_day_3 = start_today + timedelta(days=3)
 
-        date_category = "none"
-
-        if datetime:
-            datetime = datetime.astimezone(pytz.UTC)
-            if datetime < start_yesterday:
-                date_category = "before"
-            elif datetime >= start_yesterday and datetime < start_today:
-                date_category = "yesterday"
-            elif datetime >= start_today and datetime < start_day_1:
-                date_category = "today"
-            elif datetime >= start_day_1 and datetime < start_day_2:
-                date_category = "day_1"
-            elif datetime >= start_day_2 and datetime < start_day_3:
-                date_category = "day_2"
-            else:
-                date_category = "after"
+        date_category = ""
+
+        datetime = datetime.astimezone(pytz.UTC)
```

Comment:
I'd rather keep the method being able to handle falsy values :confused: 
Imho changing the initial `date_cateogry` to an empty string is a good idea, but I'd check if there's a `date_category` in `_compute_kanban_dashboard_graph` before trying to add it to the dict values instead.

Also, could you update the docstring of the method as well, since you're changing its return value? Thanks! :slightly_smiling_face:

Reply:
done, this looks better with minimum diff